eth/v1alpha1/validator.proto

```diff
@@ -97,13 +97,13 @@ service BeaconNodeValidator {
 	// The request can specify either a validator's public key or validator index.
 	// 
 	// The status response can be one of the following:
-	//	PENDING_ACTIVE - validator is waiting to get activated.
+	//	DEPOSITED - validator's deposit has been recognized by Ethereum 1, not yet recognized by Ethereum 2.
+	//	PENDING - validator is in Ethereum 2's activation queue.
 	//	ACTIVE - validator is active.
-	//	INITIATED_EXIT - validator has initiated an an exit request.
-	//	WITHDRAWABLE - validator's deposit can be withdrawn after lock up period.
-	//	EXITED - validator has exited, means the deposit has been withdrawn.
-	//	EXITED_SLASHED - validator was forcefully exited due to slashing. 
-	//	UNKNOWN - validator does not have a known status in the network.
+	//	EXITING - validator has initiated an an exit request, or has dropped below the ejection balance and is being kicked out.
+	//	EXITED - validator is no longer validating.
+	//	SLASHING - validator has been kicked out due to meeting a slashing condition.
```

Comment:
wouldnt it be better as `slashed` vs `slashing` ?

Reply:
Slashed sounds final, whereas slashing a state analagous to exiting where it is still in movement.

I did try slashed for a while, but my biggest problem with it was that you can be in the state but yet to have a final punishment applied.  From a user perspective, slashed sounds like it's done and dusted and I didn't want to give that impression as they could then be upset when what they see as a further punishment is applied.  Slashing makes it clearer that it is on-going.